saleor/dashboard/templatetags/utils.py

```diff
@@ -149,3 +150,10 @@ def sorting_header(context, field, label, is_wide=False):
         'url': '%s?%s' % (request.path, request_get.urlencode()),
         'is_active': is_active, 'sorting_icon': sorting_icon, 'label': label,
         'is_wide': is_wide}
+
+
+@register.simple_tag
+def page_published(page):
+    today = datetime.date.today()
+    return page.is_visible and \
```

Comment:
To be consistent with the rest of the codebase, you should rather break the long lines by wrapping the whole line into parenthesis instead of using backslashes:

```python
return (
    page.is_visible and (
        page.available_on is None or page.available_on <= today))
```

Reply:
I think the current PEP8 would rather recommend (notice where `and` goes):

```python
return (
    page.is_visible
    and (page.available_on is None or page.available_on <= today))
```